src/main/java/gov/usgs/locator/Event.java

```diff
@@ -537,6 +552,15 @@ public void input(LocInput in) {
     }
 
     useDecorrelation = in.UseSVD; // True when noSvd is false
+
+    // optional parameter so give it a default
+    if (in.ReassessInitialPhaseIDs != null) {
+      reassessInitialPhaseIDs =
+          in.ReassessInitialPhaseIDs; // True when ReassessInitialPhaseIDs is true
+    } else {
+      reassessInitialPhaseIDs = true;
```

Comment:
seemed like in other code the default value was false rather than true, just checking which should it be?

Reply:
This is me balancing what ray wanted (and our initial testing setup) which has the default value as false, with what Will wants in the future (once he does what he wants) which is initial phase reID disabled.